src/loader.js

```diff
@@ -252,6 +224,32 @@
     };
   }
 
+  // 4.4.5
+  Registry.prototype.uninstall = function(key) {
+    if (typeof this !== 'object')
+      throw new TypeError('Registry must be an object');
+    var entry = this._registry.registryData[key];
+    if (!entry)
+      throw new TypeError('Module ' + key + ' does not exist');
+    var stageEntry = getCurrentStage(entry);
+    if (stageEntry.stage !== 'link' && stageEntry.stage !== 'ready')
+      throw new TypeError('Module is still loading');
```

Comment:
Ah I misread which line your previous comment was on, and so I updated the wrong line. Will fix